### ISSUE TITLE
Upgrade to Version: 2.7.6 caused an error 500 

### ISSUE THREAD

**Issue by markowj** [rupok/essential-addons-for-elementor-lite#75]
I upgraded from Version: 2.7.5 to Version: 2.7.6 and received an error 500 message.  Had WordPress reboot my site and they confirmed the offending plugin was essential-addons-for-elementor-lite.  (Latest version from this morning August 17).

My system info:

Server Environment |   |  
-- | -- | --
Operating System: | Linux |  
Software: | nginx |  
MySQL version: | 5.5.5 |  
PHP Version: | 7.0.31 |  
PHP Max Input Vars: | 6144 |  
PHP Max Post Size: | 2047M |  
GD Installed: | Yes |  
ZIP Installed: | Yes |  
Write Permissions: | There are some writing permissions issues with the following directories/files: 		 - WordPress root directory |  
Elementor Library: | Connected |

**Comment by rupok** [comment]
Hi @markowj 

Your server environment seems good enough and should not face 500 error. Can you still check the allocated memory from Elementor > System Info?

Thanks!

**Comment by lauras** [comment]
I'm also on .com, and I'm facing the same problem since I first tried to install the prior release (#66). Here's what I have allocated:

```
Version: | 4.9.8 |  
WP Multisite: | No |  
Max Upload Size: | 2 GB |  
Memory limit: | 256M
```

**Comment by markowj** [comment]
I've also listed my memory (same as Laura's).  I'm not prepared to upgrade to 2.7.6 until there is confirmation that this plugin is working as it's expected.

Version: | 4.9.8 |  
WP Multisite: | No |  
Max Upload Size: | 2 GB |  
Memory limit: | 256M

**Comment by markowj** [comment]
FYI I tried to upgrade the main Elementor plugin tonight and received an Error 500 again.  I'm on WordPress.com.  Tonight they again confirmed that it is the Essentials Add-on-Lite plugin causing the issue and I've attached more details here.  Previously, I had tried to upgrade the Essentials Add-on plugin 2 days ago and received the Error 500 then too.

This is the error message.

PHP Fatal error:  Class 'Elementor\Group_Control_Base' not found in /opt/sites/1066232/wpcom-136798097/wp-content/plugins/essential-addons-for-elementor-lite/includes/elementor-helper.php on line 759
